### PR TITLE
Features: set captured image resolution, set camera target, set camera position 

### PR DESCRIPTION
**Pull request by ManifoldFR**
This follows the PR https://github.com/rdeits/meshcat/pull/111. I expose the following new features from that PR:

+ setting the camera target
+ specifying the `get_image()` resolution

Also: 
+ a convenience function for setting camera position (in right-handed coordinates)

### CODE REVIEW

File: src/meshcat/commands.py [hunk 1]
```diff
@@ -44,12 +44,35 @@ def lower(self):
             u"matrix": list(self.matrix.T.flatten())
         }
 
-class CaptureImage:
+
+class SetCamTarget:
+    __slots__ = ["value"]
+    def __init__(self, value):
```
Comment (rdeits) [inline src/meshcat/commands.py:50]: `value` is pretty vague as a variable name. What are the expected contents, and what do they do?

Comment (ManifoldFR) [reply src/meshcat/commands.py]: It's supposed to be the camera's target position, in 3D. I made this more explicit the commit I pushed just now.

**Comment by ManifoldFR** [comment]
Just rebased on master.

**Comment by ManifoldFR** [comment]
Did a clean rebase on master and updated `demo.ipynb` with examples of the camera pos/target options

**Comment by ManifoldFR** [comment]
Hi @rdeits, sorry to be a bother, but do you think this PR is ready to be merged?
We kind of let it marinate in open PR limbo (😅) but now with @jcarpent we're thinking about using the new features from this PR in a future release of Pinocchio

Should I work on the code coverage or is that OK?

**Comment by rdeits** [comment]
I'm embarrassed that it took me so long to get back to this, but I do really appreciate the contribution. I've verified that this works well locally, so let's do it!

**Pull request merged**